test/api.js

```diff
@@ -750,7 +757,7 @@ describe('/v1', function() {
           });
         }
 
-        it('should get auth token with secret', function(done){
+        it('should get auth token with secret', function(){
           return getCode(clientId).then(function(code) {
```

Comment:
Aha, no, what used to get returned here was the result of `.done(done, done)` which is not a promise.